### PR TITLE
Adds support for paging through entities collection

### PR DESCRIPTION
**Pull request by irvinesunday**
Closes https://github.com/microsoftgraph/microsoft-graph-explorer-api/issues/199

Proposes:
- Adding x-ms-pageable extension to allow for paging of collections.
- Extend the current test to include the above added extension.

### CODE REVIEW

File: src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs [hunk 1]
```diff
@@ -59,5 +59,10 @@ internal static class Constants
         /// extension for group type
         /// </summary>
         public static string xMsDosGroupPath = "x-ms-docs-grouped-path";
+
+        /// <summary>
+        /// extension for paging
+        /// </summary>
+        public static string xMsPageable = "x-ms-pageable";
```
Comment (xuzhg) [inline src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs]: Can you add a config for this extension, so that the existing customer is not affected

Comment (xuzhg) [reply src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs]: #Resolved.

**Comment by xuzhg** [comment]
Looks good to me. 🚢

**Comment by darrelmiller** [comment]
@irvinesunday   This PR has failing tests.  Can you make the necessary changes so the tests pass?  Thanks.

**Comment by irvinesunday** [comment]
@darrelmiller This is closed; did you mean this PR? https://github.com/microsoft/OpenAPI.NET.OData/pull/55

**Pull request merged**